site/examples/navigation.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const menuNavigation = `<nav'>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3 is-active' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-home'/></svg> Home
+  </a>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-user'/></svg> Account
+  </a>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-logout'/></svg> Logout
+  </a>
+</nav>`;
+
+const underlinedTabs = `<div class='bg-gray border-b border--black flex-parent flex-parent--stretch-cross h40 txt-bold txt-s'>
+  <button class='flex-child mb-neg1 px12 py6 border-b border--black border--white-on-active color-lighten50 color-white-on-active color-lighten75-on-hover'>One</button>
+  <button class='flex-child mb-neg1 px12 py6 border-b border--black border--white-on-active color-lighten50 color-white-on-active color-lighten75-on-hover is-active'>Active</button>
+  <button class='flex-child mb-neg1 px12 py6 border-b border--black border--white-on-active color-lighten50 color-white-on-active color-lighten75-on-hover'>Two</button>
+</div>
+<div class='bg-gray-dark color-white p12'>
+  Changing content
+</div>`;
+
+const breadcrumbs = `<div class='flex-parent flex-parent--stretch-cross txt-s'>
+  <a href='#' class='flex-child link link--darken50'>One</a>
+  <svg class='icon flex-child align-middle'><use xlink:href='#icon-caret-right'/></svg>
```

Comment:
@davidtheclark removing `align-middle` sounds good.

> `align-middle` should be tossed out in favor of flexbox vertical centering.

Isn't that what `flex-parent--center-cross` contributes?